index.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html>
-	<head>
-		<meta charset="utf-8">
-		<title>Welcome to Potionfy</title>
-	</head>
-	<body>
-		
-	</body>
+  <head>
+    <meta charset="utf-8" />
+    <title>Welcome to Potionfy</title>
+  </head>
+  <body>
+    <h1>Welcome to Pontionfy! Enjoy your stay.</h1>
```

Comment:
Could you remove the 'Enjoy your stay.' string and make the 'Welcome to potionfy!' red?

Reply:
Sure, can do!